cinema/serializers.py

```diff
@@ -1 +1,103 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(source="__str__", read_only=True)
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    capacity = serializers.SerializerMethodField(method_name="get_capacity")
```

Comment:
The `capacity` field is defined using `SerializerMethodField`, which requires a method `get_capacity` to calculate or retrieve the value. Ensure that the `CinemaHall` model has a `capacity` attribute or modify `get_capacity` to compute the capacity based on `rows` and `seats_in_row`.